CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+2017-10-05 Hoat Le <hoatle@teracy.com>
+--------------------------------------
+
+    Version: 0.14.0
+
+    - Features:
+        * Type hints support for py3 #34
+        * `base_class` introduced #38
+        * Add options passing to `werkzeug.Routing.Rule` from register function #46
+        * Enable using custom decorators in FlaskView #29
+        * Should allow specifying excluded methods from becoming routes #41
+
+    - Improvements:
+        * Better representations #33 #72
+        * Docker workflow along with teracy-dev for better dev setup #32 #63 #66 #67
+
+    - Bugfixes:
+        * The behavior of the trailing slash should be more intuitive and consistent #47
+        * `base_args` should not be overriden when `route_base` is set #50
+
+    - Tasks:
+        * Keep the original license #51
+        * Fix docs publishing #61
+        * Breaking change re: FlaskView.decorators missing in CHANGELOG (for v0.11.0) #49
+        * Should update docs for v0.14.0 #60 #73
```

Comment:
I don't think this line needs to be included, as it should be obvious.

Reply:
removed